tools/edumfa-migrate-linotp.py

```diff
@@ -412,7 +414,7 @@ def insert_chunks(conn, table, values, chunk_size=100000, record_name="records")
 
             key_enc = r['LinOtpKeyEnc']
             key_iv = r['LinOtpKeyIV']
-            if config_obj.REENCRYPT:
+            if config_obj.REENCRYPT and key_enc != "" and key_iv != "":
```

Comment:
If REENCRYPT is turned on, the script will fail on token, that have no encryption. Like "yubico". They just have an id which can not be reencrypted.

Reply:
Looks fine for me